subprojects/gradle/cd/src/gradleTest/kotlin/com/avito/ci/steps/TestSummaryStepTest.kt

```diff
@@ -29,24 +30,62 @@ internal class TestSummaryStepTest {
             dryRun = true
         )
             .assertThat()
-            .buildSuccessful()
-            .tasksShouldBeTriggered(
-                ":app:instrumentationUi",
-                ":app:testSummary",
-                ":app:fullCheck"
-            )
-            .inOrder()
+            .buildSuccessful().run {
+                tasksShouldBeTriggered(
+                    ":app1:instrumentationUi",
+                    ":testSummarySomePlanSomeJob",
+                    ":app1:fullCheck"
+                ).inOrder()
+
+                tasksShouldBeTriggered(
```

Comment:
Is the same check called twice here?

Reply:
It is for different apps - "app1", "app2"

Reply:
ah, didn't notice